Storage/tests/Unit/Connection/RestTest.php

```diff
@@ -204,7 +205,8 @@ public function testInsertObject(
         array $options,
         $expectedUploaderType,
         $expectedContentType,
-        array $expectedMetadata
+        array $expectedMetadata,
+        array $metadataDoesNotHave = []
```

Comment:
```suggestion
        array $metadataKeysWhichShouldNotBetSet = []
```

I know it is a bit verbose, but the current naming wasn't reading very clearly to me :). Does this still fall in line with what your expectation is?

Reply:
works for me!